src/sql/src/vars.rs

```diff
@@ -1169,6 +1181,9 @@ pub struct SystemVars {
     // features
     enable_with_mutually_recursive: SystemVar<bool>,
 
+    // storage configuration
+    enable_multi_worker_storage_persist_sink: SystemVar<bool>,
```

Comment:
Any specific reason to create a new category instead of adding it to `features`?

Reply:
its not a sql feature!